core/src/main/java/org/axonframework/serialization/MessageSerializer.java

```diff
@@ -85,9 +82,6 @@ public MessageSerializer(Serializer serializer) {
      */
     public static <T> SerializedObject<T> serializeMetaData(Message<?> message, Serializer serializer,
                                                             Class<T> expectedRepresentation) {
-        if (message instanceof SerializationAware) {
```

Comment:
As the `Message` is now `SerializationAware`, shouldn't we simply return `message.serializeMetaData(serializer, expectedRepresentation)` directly?